linera-views/src/common.rs

```diff
@@ -38,6 +41,41 @@ pub(crate) enum Update<T> {
     Set(T),
 }
 
+/// Status of a table at the creation time of a [`KeyValueStoreClient`] instance.
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum TableStatus {
+    /// Table was created during the construction of the [`KeyValueStoreClient`] instance.
+    New,
+    /// Table already existed when the [`KeyValueStoreClient`] instance was created.
+    Existing,
+}
+
+#[cfg(any(test, feature = "test"))]
+static TEST_COUNTER: Lazy<Arc<RwLock<u32>>> = Lazy::new(|| Arc::new(RwLock::new(0)));
+
+/// Returns a unique table name for testing.
+#[cfg(any(test, feature = "test"))]
+pub async fn get_table_name() -> String {
+    let mut counter = TEST_COUNTER.write().await;
+    *counter += 1;
+    format!("test_table_{}", *counter)
+}
+
+/// The common initialization parameters for the KeyValueStore
+#[derive(Debug, Clone)]
+pub struct CommonStoreInit {
```

Comment:
Maybe `StoreConfig` or `InitConfig`?

Reply:
Still, it should end with `Config` as we do in the rest of the code.

Reply:
Ok, so `CommonStoreConfig`?